modules/custom/cu_core/cu_core.install

```diff
@@ -72,6 +72,7 @@ function cu_core_install() {
   variable_set('express_help_base_url', 'http://www.colorado.edu/webcentral');
 
   // Set message for when enabling of bundles is turned off.
```

Comment:
I don't think we should be changing the variable in code, but not updating the message text in code as well.  We should either put both in code or do both with drush commands, not both.